### ISSUE TITLE
Python - DLL load failed on import

### ISSUE THREAD

**Issue by Fladenbro** [flatironinstitute/finufft#184]
Hi,

thanks for providing this awesome code!

I'm struggling to use finufft within my Python code on Windows 10, though.
The installation seems to work fine via pip since it returns success. However, importing from finufft raises the following error:
```
from finufft import Plan
  File "D:\...\venv\lib\site-packages\finufft\__init__.py", line 15, in <module>
from finufft._interfaces import Plan
  File "D:\...\venv\lib\site-packages\finufft\_interfaces.py", line 12, in <module>
import finufft._finufft as _finufft
ImportError: DLL load failed while importing _finufft: The specified module could not be found.
```

I couldn't find out what DLL is missing or what went wrong upon installation. Re-installation via pip doesn't help. I haven't experienced this with any other pip installation so far...

Is there any way I can fix this?
I'm extremely grateful for any piece of advice!
Thanks

--
Windows 10 Enterprise Version 1709 x64
PyCharm Community Edition 2021.1.1
Python 3.8.3
pip 21.1
finufft 2.0.0

**Comment by lu1and10** [comment]
Hi,

The windows python wheels uploaded to pypi are not repaired(in mac we use
delocate-wheel, in linux we use  auditwheel to repair the wheel including
dependent binary libs).
But for windows the wheels are not repaired, so some DLL is missing, one
may need to install dependent binary libs themselves(fftw and gcc libs).

**Comment by Fladenbro** [comment]
Hi,

thanks for the quick response. I tried to get finufft up and running by building everything from scratch. Due to my inexperience I couldn‘t make it work this way, unfortunately.

However, the installation worked perfectly fine at a friend‘s python setup on windows via the pip command without further ado. We compared our setups and found that the major difference was our Python environment. I had everything set up in a VirtualEnvironment (virtualenv) directly in PyCharm while he used Anaconda. 

I ended up installing Anaconda and creating a new Conda-environment. To my surprise, when I used pip install finufft in this environment in PyCharm, it all worked fine and I‘m now able to use finufft in all its beauty. 

I can‘t exactly tell why this is working. I can only guess that an Conda-environment comes with pre-installed libraries that are needed to integrate finufft.

Hope this helps to save someone else‘s time or to improve the installation process on windows. 

Cheers
